src/metamodels/crosscat.py

```diff
@@ -228,15 +228,31 @@ class CrosscatMetamodel(metamodel.IBayesDBMetamodel):
     with names that begin with ``bayesdb_crosscat_``.
 
     `crosscat` can be a crosscat engine instance or constructor. Prefer to pass
-    a constructor, since that way the engine obtains a random seed from the
-    BayesDB object. The 'crosscat as instance' case is kept for backwards
-    compatibility.
+    a constructor, since that way a new crosscat engine is constructed for each
+    query, and its random seed is taken from the BayesDB object. The 'crosscat
+    as instance' case is kept for backwards compatibility.
```

Comment:
People's existing bayeslite code will break painfully.  I don't think we have to stay backwards compatible forever, but it seems considerate to keep the old behavior for a while.